agent/provider/src/startup_config.rs

```diff
@@ -153,8 +208,8 @@ pub struct ReceiverAccount {
     #[structopt(long, env = "YA_ACCOUNT")]
     pub account: Option<NodeId>,
     /// Payment network.
-    #[structopt(long = "payment-network", env = "YA_PAYMENT_NETWORK", possible_values = NetworkName::VARIANTS, default_value = NetworkName::Mainnet.into())]
-    pub networks: Vec<NetworkName>,
+    #[structopt(long = "payment-network", env = "YA_PAYMENT_NETWORK", default_value = NetworkName::Mainnet.into())]
```

Comment:
Is it possible to provide these `possible_values`?
They were very helpful.
If it isn't, than maybe at least description saying what can be passed here?

Reply:
Unfortunately, no, it can't be generates as before, mostly due to API constraints of structopt. I did add a help message